src/index.html

```diff
@@ -10,6 +10,8 @@
   <script type="text/javascript" src="http://code.jquery.com/jquery-1.9.1.min.js" charset="utf-8"></script>
```

Comment:
Can you update to jQuery 3, please?
(but make sure, the app still works. I don't know if Angular is using it in the background.)
https://blog.jquery.com/2016/06/09/jquery-3-0-final-released/